README.rst

```diff
@@ -54,21 +37,50 @@ is an example URL conf::
         url(r'^secret/', decorator_include(login_required, 'mysite.secret.urls')),
     ]
 
-Running tests
--------------
 
-If you are in a fresh virtualenv to work on ``decorator_include``, install the
-Django version you want::
+Supported versions
+------------------
+
+============== ==================
+Django version Python versions
+============== ==================
+1.11           2.7, 3.4, 3.5, 3.6
+2.0            3.4, 3.5, 3.6
+============== ==================
+
+All library versions to use for old django/python support
```

Comment:
I would capitalize "Django" and "Python" here. They are proper nouns.

Reply:
Done